src/chainparams.h

```diff
@@ -56,7 +56,8 @@ class CChainParams
     const Consensus::Params& GetConsensus() const { return consensus; }
     const CMessageHeader::MessageStartChars& MessageStart() const { return pchMessageStart; }
     int GetDefaultPort() const { return nDefaultPort; }
-
+    /** This chain can have buried and bip9 soft fork parameters overridden with -bip9params and -buriedsfparams */
+    bool AllowsOverriddenSoftFork() const { return fAllowsOverriddenSoftFork; }
```

Comment:
Is the getter necessary? Params() already returns a const ref, why not use Params().fAllowsOverriddenSoftFork directly in init (making fAllowsOverriddenSoftFork public)?